pkg/api/controller.go

```
@@ -1884,36 +1884,54 @@ func (c *Controller) GetStorageConfig(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	// TODO (gilo): is StorageID relevant here?
-	storageConfig, err := c.getStorageConfig("")
-	if c.handleAPIError(r.Context(), w, r, err) {
-		return
-	}
-
-	writeResponse(w, r, http.StatusOK, storageConfig)
+	storageCfg, _ := c.getStorageConfig(config.SingleBlockstoreID)
```

Is it ok that we swallowed the error here?